tests/components/smartthings/test_lock.py

```diff
@@ -110,16 +95,3 @@ def test_is_lock(device_factory):
     state = hass.states.get('lock.lock_1')
     assert state is not None
     assert state.state == 'locked'
-
-
-async def test_unload_config_entry(hass, device_factory):
```

Comment:
Why did you remove this test?

Reply:
Mentioned above, It looks like base `lock` component in HA doesn't support `async_unload_entry`.

Reply:
I think you found a bug in the lock component!  I'll add it in a seperate PR (or you're welcome to)

The code is simply the same as all the other components:
```python
async def async_unload_entry(hass, entry):
    """Unload a config entry."""
    return await hass.data[DOMAIN].async_unload_entry(entry)
```

Reply:
I'll take care of it!  don't worry about it for now, but we do need that support for the SmartThings component.

Reply:
Just tested and looks like all works once added so raised a PR #21027, guess we will have to wait for that PR to get merged before the test here will pass 😄 Will add the test back in.